src/Polysemy/Lift/Type.hs

```diff
@@ -2,7 +2,10 @@
 
 {-# OPTIONS_HADDOCK not-home #-}
```

Comment:
Ah, this isn't supposed to be here anymore, I think!

Reply:
Nevermind, it actually should, I guess, because of the reexports. I need to improve my haddock-fu.